src/main/java/edu/cornell/kfs/module/ar/CuArPropertyConstants.java

```diff
@@ -1,11 +1,23 @@
 package edu.cornell.kfs.module.ar;
 
+import org.kuali.kfs.module.ar.ArPropertyConstants;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.kfs.sys.KFSPropertyConstants;
+
 public class CuArPropertyConstants {
     
     public static class ContractsAndGrantsBillingAwardFields {
         public static final String FINAL_BILL = "finalBill";
         public static final String PARTIAL_BILL = "partialBill";
         public static final String TOTAL_PROGRAM_OUTLAYS_TO_DATE = "totalProgramOutlaysToDate";
+        public static final String ACCOUNT_DETAILS_CHART_OF_ACCOUNTS_CODE = ArPropertyConstants.ACCOUNT_DETAILS
+                + KFSConstants.DELIMITER + KFSPropertyConstants.CHART_OF_ACCOUNTS_CODE;
+        public static final String ACCOUNT_DETAILS_CONTRACT_CONTROL_CHART_OF_ACCOUNTS_CODE = ArPropertyConstants.ACCOUNT_DETAILS
+                + KFSConstants.DELIMITER + KFSPropertyConstants.ACCOUNT
+                + KFSConstants.DELIMITER + KFSPropertyConstants.CONTRACT_CONTROL_CHART_OF_ACCOUNTS_CODE;
+        public static final String ACCOUNT_DETAILS_CONTRACT_CONTROL_ACCOUNT_NUMBER = ArPropertyConstants.ACCOUNT_DETAILS
```

Comment:
These variable names don't seem to represent the values being concatenated together.

Reply:
After further consideration, I think I can improve the variable names by introducing a new nested class to CuArPropertyConstants and putting the variables in there. I'll work on restructuring these variables/constants accordingly.